packages/ds-healthcare-gov/package.json

```diff
@@ -20,10 +20,10 @@
     "src"
   ],
   "dependencies": {
-    "@cmsgov/design-system": "^2.13.0",
+    "@cmsgov/design-system": "^3.0.0-beta.1",
+    "@types/js-cookie": "^2.2.7",
```

Comment:
Was this needed as part of the version bump?

Reply:
it's not a bump, the dependency name got moved around for unknown reasons. In the previous version of the file, you can see it on line 26